packages/server/src/models/shelf-invitation.entity.ts

```diff
@@ -0,0 +1,19 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity()
+export default class ShelfInvitation {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  userEmail: string;
+
+  @Column()
+  shelfId: number;
```

Comment:
I think shelfId in Invitation should be connected to a Shelf id also

Reply:
We kept the temporary tables simple, e.g. `email-reset`, `email-verification`, `password-reset` all have a simple `userId: number` column, which kind of makes sense, because all these are temporary tables, which should be purged on a regular basis (token expiry is 12h). So I made this one simple too.
But OK, let's try to make it the right way this time, added a `ManyToOne` relation with `onDelete: CASCADE`.
Fixed.